### PR TITLE
[5.10.0] API doc link updates

### PR DESCRIPTION
**Pull request by divyaamunugama**

### CODE REVIEW

File: en/docs/develop/using-the-account-recovery-rest-apis.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 !!! tip     
     For information on password and username recovery using REST APIs, see the [REST API swagger docs on account
-    recovery](https://docs.wso2.com/display/IS510/apidocs/account-recovery/).
+    recovery](https://api-docs.wso2.com/apidocs/is/is510/account-recovery/).
```
Comment (divyaamunugama) [inline en/docs/develop/using-the-account-recovery-rest-apis.md:5]: Check this link

File: en/docs/develop/using-the-consent-management-rest-apis.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ Add the following configuration to the `<IS_HOME>/repository/conf/deployment.tom
 
 !!! info
     
-    For information on the REST APIs, supported operations and sample requests/responses, see [Consent Management APIs Swagger Documentation](https://docs.wso2.com/display/IS510/apidocs/Consent-management-apis/).
+    For information on the REST APIs, supported operations and sample requests/responses, see [Consent Management APIs Swagger Documentation](https://api-docs.wso2.com/apidocs/is/is510/Consent-management-apis/).
```
Comment (divyaamunugama) [inline en/docs/develop/using-the-consent-management-rest-apis.md:69]:
```suggestion
    For information on the REST APIs, supported operations and sample requests/responses, see [Consent Management APIs Swagger Documentation](https://api-docs.wso2.com/apidocs/is/is510/consent-management/).
```

File: en/docs/develop/using-the-openid-connect-dynamic-client-registration-rest-apis.md [hunk 3]
```diff
@@ -5,7 +5,7 @@
     For information on OAuth2 - OpenID Connect dynamic client registration
     and how to manage OAuth2 applications using REST APIs, see the [REST API
     swagger docs on OAuth2 Dynamic Client
-    Registration](https://docs.wso2.com/display/IS510/apidocs/OAuth2-dynamic-client-registration).
+    Registration](https://api-docs.wso2.com/apidocs/is/is510/OAuth2-dynamic-client-registration).
```
Comment (divyaamunugama) [inline en/docs/develop/using-the-openid-connect-dynamic-client-registration-rest-apis.md:8]:
```suggestion
    Registration](https://api-docs.wso2.com/apidocs/is/is510/dynamic-client-regisration).
```

File: en/docs/learn/resending-account-recovery-confirmation-emails.md [hunk 4]
```diff
@@ -355,7 +355,7 @@ Follow the steps below to configure WSO2 Identity Server to resend account recov
     6.  **Email Footer** :  --
 
 !!! tip
-    For information on the REST API for resending confirmation code, see [Resend-Code Using REST API](https://docs.wso2.com/display/IS590/apidocs/self-registration/#!/operations#ResendConfirmation#resendCodePost).
+    For information on the REST API for resending confirmation code, see [Resend-Code Using REST API](https://api-docs.wso2.com/apidocs/is/is590/self-registration/#!/operations#ResendConfirmation#resendCodePost).
```
Comment (divyaamunugama) [inline en/docs/learn/resending-account-recovery-confirmation-emails.md:358]:
```suggestion
    For information on the REST API for resending confirmation code, see [Resend-Code Using REST API](https://api-docs.wso2.com/apidocs/is/is590/self-registration/#!/operations#SelfRegister#resendCodePost).
```

File: en/docs/develop/using-the-account-recovery-rest-apis.md (hunk 1 above)
Comment (divyaamunugama) [reply en/docs/develop/using-the-account-recovery-rest-apis.md:5]: Replacing this with 5.9.0 API as the 5.10.0 API is not found.

Comment (divyaamunugama) [inline en/docs/develop/using-the-account-recovery-rest-apis.md:5]:
```suggestion
    recovery](https://api-docs.wso2.com/apidocs/is/is590/account-recovery-v5.9.0/).
```

**Review by nilminiwso2 — APPROVED** [review]

**Review by himeshsiriwardana — APPROVED** [review]

**Pull request merged**